tests/cpp/test_gpu_tensorcore.cpp

```diff
@@ -52,11 +52,22 @@
 #include "c10/core/ScalarType.h"
 
 namespace nvfuser {
+namespace {
+class GPUTTensorCoreTest : public NVFuserTest {
+ protected:
+  GPUTTensorCoreTest() {
+    DisableOptionsGuard::getCurOptions().set(DisableOption::MatmulExprEval);
+  }
+
+ private:
+  DisableOptionsGuard opt_guard_;
```

Comment:
A comment might be useful here, because at a glance it looks like this is an unused variable. Instead, it's important that this is default initialized before the constructor body is run so that `DisableOptionsGuard::getCurOptions()` refers to the new options and not the previous ones, which are held in `options_guard_`.